### ISSUE TITLE
[Bug]: After deleting an OpenAI-API-Compatible model you cannot add a new model

### ISSUE THREAD

**Issue by rplescia** [infiniflow/ragflow#2058]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Branch name

0.9.0

### Commit ID

na

### Other environment information

```Markdown
Hardware: Cisco UCS240 M5
OS: Ubuntu 24.04
```

### Actual behavior

After adding an OpenAI-API-Compatible model I wanted to change the configuration to switch from vLLM to Llamafile. To do this I needed to remove the vLLM model and add the Llamafile configuration. After deleting the model I click on the OpenAI-API-Compatible provider, instead of getting a dialog box with all the options (base URL, model name, key etc.) it only asks to modify the API key.

![image](https://github.com/user-attachments/assets/a747b0eb-5ee6-4c45-840a-4632aa589872)

### Expected behavior

Show full dialog box to add new OpenAI-API compatible endpoint

### Steps to reproduce

```Markdown
Add an OpenAI-API-Compatible model, delete it, and try adding a new one.
```

**Comment by hangters** [comment]
hi  @rplescia   Thank you for pointing out this issue. I tried adding and then deleting a model in the OpenAI-API-Compatible, and after clicking on it again, everything worked fine. I suggest pulling the latest Docker image to see if that resolves the issue. If the problem persists, please let me know.

**Comment by rplescia** [comment]
Hi, I can confirm this bug has been fixed in v0.10.0. Thanks!
